crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -76,7 +82,7 @@ impl<'a> Commands<'a> {
     /// }
     /// # example_system.system();
     /// ```
-    pub fn spawn(&mut self) -> EntityCommands<'a, '_> {
+    pub fn spawn<'b>(&'b mut self) -> EntityCommands<'a, 'b> {
```

Comment:
I would honestly appreciate more verbose lifetime names, but that may be out of scope.